packages/scoutgame/src/builders/aggregateTokensPurchased.ts

```diff
@@ -0,0 +1,12 @@
+import { prisma } from '@charmverse/core/prisma-client';
```

Comment:
thanks for adding this to a lib. NBD but i think it would make more sense inside a scouts folder